src/tool_help.c

```diff
@@ -65,6 +65,7 @@ static const char *const helptext[] = {
   "     --crlf          Convert LF to CRLF in upload",
   "     --crlfile FILE  Get a CRL list in PEM format from the given file",
   " -d, --data DATA     HTTP POST data (H)",
+  "     --data-raw      HTTP POST data, '@' allowed (H)",
```

Comment:
suggestions for you

```
 "     --data-raw DATA  HTTP POST data, '@' allowed (H)",
```
And making a separate entry for --data-raw in curl.1 and make the last line in your description (Added in 7.43.0).
You could even refer to `--data` like this
```
.IP "--data-raw <data>"
Almost the same as \fI--data\fP but does not have a special interpretation of
the @ character. See \fI-d, --data\fP.

(Added in 7.43.0)
```
Also I'd edit the `--data` option so that it says something about `--data-raw`:
```
\fI-d, --data\fP is the same as \fI--data-ascii\fP. \fI--data-raw\fP is almost
the same but does not have a special interpretation of the @ character. To post
data purely binary, you should instead use the \fI--data-binary\fP option. To
URL-encode the value of a form field you may use \fI--data-urlencode\fP.
```
then modify the last paragraph and add something about data raw for example:
```
If you start the data with the letter @, the rest should be a file name to
read the data from, or - if you want curl to read the data from
stdin. Multiple files can also be specified. Posting data from a file
named 'foobar' would thus be done with \fI--data\fP @foobar. When --data is
told to read from a file like that, carriage returns and newlines will be
stripped out. If you don't want the @ character to have a special
interpretation use \fI--data-raw\fP instead.
```
If you are going to take on documenting it in curl.1 please wrap at max length 79 same as source